main.tf

```diff
@@ -11,6 +11,9 @@ locals {
   ipv4_cidr_block_associations = local.enabled ? (
     length(local.additional_cidr_blocks_map) > 0 ? local.additional_cidr_blocks_map : var.ipv4_additional_cidr_block_associations
   ) : {}
+  default_adoption_tags = merge(module.label.tags, {
+    Name = join(module.label.delimiter, [module.label.id, "default"])
+  })
 }
 
 module "label" {
```

Comment:
It's odd that this label does absolutely nothing but has been created.

If we added `attributes = ["default"]` here, we can set the below tags to be `module.label.tags` and remove the local above.

Reply:
Hmm, but it could cause other issues (such as recreations of resources i.e. no backward compatibility) depending on where this label is used...